idseq_pipeline/commands/common.py

```diff
@@ -491,36 +494,37 @@ def fetch_from_s3(source,
             # It's okay if the parent directory already exists, but all other errors are fatal.
             if e.errno != os.errno.EEXIST:
                 raise
-        with iostream:
-            try:
-                if allow_s3mi:
+        with iostream_uploads:  # Limit concurrent uploads so as not to stall the pipeline.
```

Comment:
Not here.   This function does downloads, not uploads.

Reply:
fixed thanks